api/api/models/media.py

```diff
@@ -275,17 +275,14 @@ def _perform_index_update(self, method: str, raise_errors: bool, **es_method_arg
                     refresh=True,
                     **es_method_args,
                 )
-            except (NotFoundError, BadRequestError) as e:
-                if isinstance(e, NotFoundError):
-                    # This is expected for the filtered index, but we should still
-                    # log, just in case.
-                    logger.warning(
-                        f"Document with _id {document_id} not found "
-                        f"in {index} index. No update performed."
-                    )
-                    continue
-                else:
-                    raise e
+            except NotFoundError:
+                # This is expected for the filtered index, but we should still
+                # log, just in case.
+                logger.warning(
+                    f"Document with _id {document_id} not found "
+                    f"in {index} index. No update performed."
+                )
+                continue
```

Comment:
Super-nit: I also realised, this `continue` is redundant here.

Reply:
I left it in to make explicit that the loop is meant to continue in spite of the 404. Do you think that's necessary? Or is there a better way to communicate that (I suppose a comment would do).

Reply:
No problem either way, we can leave it in.